Formula/beansdb.rb

```diff
@@ -3,6 +3,7 @@ class Beansdb < Formula
   homepage "https://github.com/douban/beansdb"
   url "https://github.com/douban/beansdb/archive/v0.6.tar.gz"
   sha256 "b24512862f948d5191f5c43316a41f632bc386f43dcbb69b03ffffe95122a33e"
+  revision 1
```

Comment:
Don't need a `revision` for this as we're not altering the package contents. `revision` is used to force an upgrade for all users so is only really needed for package contents changes (and even then only really widespread bug fixes).